src/Symfony/Component/HttpFoundation/Request.php

```diff
@@ -430,6 +430,8 @@ public static function create(string $uri, string $method = 'GET', array $parame
      * This is mainly useful when you need to override the Request class
      * to keep BC with an existing system. It should not be used for any
      * other purpose.
+     *
+     * @param callable():SessionInterface|null $callable
```

Comment:
:thinking: This method is about `requestFactory`. I think you confuse with the `setSessionFactory` method

Reply:
I remove this. 

`setSessionFactory()` already has this annotation.